docs/usage.rst

```diff
@@ -1,17 +1,19 @@
 Usage
 =====
 
-tedana minimally requires:
+``tedana`` minimally requires:
 
-#. acquired echo times (in milliseconds), and
-#. functional datasets equal to the number of acquired echoes.
+  #. Acquired echo times (in milliseconds)
+  #. Functional datasets equal to the number of acquired echoes
 
 But you can supply many other options, viewable with ``tedana -h`` or
 ``t2smap -h``.
 
 For most use cases, we recommend that users call tedana from within existing fMRI preprocessing
-pipelines such as `fMRIPrep`_ or `afni_proc.py`_..
-Users can also construct their own preprocessing pipelines from which to call tedana;
+pipelines such as `fMRIPrep`_ or `afni_proc.py`_.
+fMRIPrep currently supports optimal combination through ``tedana``, but not the full multi-echo
```

Comment:
Should we link to this section in our docs, on what the optimal combination is ?